packages/mui-material/src/Typography/Typography.test.js

```diff
@@ -112,6 +113,25 @@ describe('<Typography />', () => {
     });
   });
 
+  describe('prop: color', () => {
+    it('should check for invalid color value', () => {
+      const consoleWarnStub = sinon.stub(console, 'warn');
+
+      render(
+        <Typography variant="h6" color="background">
+          Hello
+        </Typography>,
+      );
+
+      sinon.assert.calledWith(
+        consoleWarnStub,
+        'MUI: The value found in theme for prop: "background" is an [Object] instead of string or number. Check if you forgot to add the correct dotted notation, eg, "background.paper" instead of "background".',
+      );
+
+      consoleWarnStub.restore();
```

Comment:
```suggestion
      expect(() => render(
        <Typography variant="h6" color="background">
          Hello
        </Typography>,
      )).toWarnDev('MUI: The value found in theme for prop: "background" is an [Object] instead of string or number. Check if you forgot to add the correct dotted notation, eg, "background.paper" instead of "background".')
```

No need to use sinon. Same change in other place as well.

Reply:
I am unable to pass any of the warn tests this way, I get the similar error for each one of them: 
```shell
<Typography /> prop: color should check for invalid color value FAILED
        AssertionError: Recorded unexpected console.warn calls: 

          - Expected no more error messages but got:
        "MUI: The value found in theme for prop: "background" is an [Object] instead of string or number. Check if you forgot to add the correct dotted notation, eg, "background.paper" instead of "background"."
```
I am unable to debug this.

Reply:
Here'e the changes I did - https://github.com/mui/material-ui/compare/master...brijeshb42:material-ui:issue-38478?expand=1

Basically, you add to call `toWarnDev([msg, msg])` expecting the warn to be called twice. That's why it was failing.

Reply:
how can I merge these changes? kindly tell if anything else remaining to work on this.

Reply:
You can just copy-paste the `toWarnDev` related changes to your branch. Nothing else after that.